docs/contribute/RFC.md

```diff
@@ -0,0 +1,39 @@
+## Goal
+
+The goal of the RFC process is to establish a consistent and controlled path for introducing new features into the project.
+
+Many changes, such as bug fixes and documentation improvements, can be implemented and reviewed via the normal GitHub pull request workflow. Some changes, however, are considered “substantial”, and we ask that these undergo a design process, solicit community input, and reach a consensus among the Storybook core team.
+
+### “Feature Request” vs “RFC”
+
+A *feature request* is a straightforward and relatively informal way for Storybook users to suggest a new feature or enhancement to the project. While feature requests can provide valuable insights and ideas, they typically do not involve an in-depth design process or require consensus among the core team. Feature requests are usually open to discussion and may or may not be implemented based on factors like popularity, feasibility, and alignment with the project's goals.
+
+On the other hand, an *RFC* is a more formalized and structured process for proposing substantial changes or additions to the project. It involves following a defined set of steps to ensure that the proposed feature or modification receives proper consideration, design, and feedback. RFCs are typically used for changes that have a significant impact on the project, such as introducing new API functionality, removing existing features, or establishing new usage conventions. The RFC process aims to foster discussions, gather feedback from a wider audience, and reach consensus among the core team before integrating the proposed change into the project. Accepted RFCs have a higher likelihood of being implemented compared to regular feature requests.
+
+## The RFC lifecycle
+
+#### 1. `Status: Proposed` 
+
+Open a new GitHub discussion in the [“RFC” category](https://github.com/storybookjs/storybook/discussions/new?category=rfc). Fill out the form as instructed. 
+
+Details matter — RFCs that do not present convincing motivation, demonstrate lack of understanding of the design's impact, or are disingenuous about the drawbacks or alternatives tend to be poorly-received.
```

Comment:
@vanessayuenn what do you think about adding some context here and inviting the author of the RFC or the person willing to implement it to the Storybook triage meeting to allow him to present his/her case? That would help us maintainers have some additional context on the RFC and get some additional context.

Reply:
That's a fantastic idea!